en/ros2/user_guide.md

```diff
@@ -751,6 +751,115 @@ will generate topics under the namespaces:
 
 :::
 
+## PX4 ROS 2 Service Servers
+
+<Badge type="tip" text="PX4 v1.15" />
+
+PX4 uXRCE-DDS middleware supports [ROS 2 services](https://docs.ros.org/en/jazzy/Concepts/Basic/About-Services.html).
+Services are remote procedure calls, from one node to another, that return a result.
+
+A service server is the entity that will accept a remote procedure request, perform some computation on it, and return the result.
+They simplify communication between ROS 2 nodes and PX4 by encapsulating and associating the request and response topics, and ensuring that replies are only returned to the specific requesting user.
+This is much easier that publishing the request, subscribing to the reply, and filtering out any unwanted responses.
```

Comment:
I pulled this out of your Vehicle command docs below, because I presume it is true in all cases. Removed text this comes from was:

> This is an helpful alternative to having to create a topic publisher for `/fmu/in/vehicle_command` and a topic subscriber for `/fmu/out/vehicle_command_ack`, and then having to filter out unwanted `VehicleCommandAck` messages.